src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -1000,6 +1002,18 @@ private CaseDTO createNewCachedCaseResponse(CachedCase newCase) throws CTPExcept
     return response;
   }
 
+  private void checkCaseTypeForNewAddress(
+      CaseType caseType, uk.gov.ons.ctp.integration.contactcentresvc.representation.Region region)
+      throws CTPException {
+    if ((caseType.name().equals("CE")) && (region.name().equals("N"))) {
+      String message =
+          "All queries relating to Communal Establishments in Northern Ireland "
+              + "should be escalated to NISRA HQ";
+      log.with(caseType.name()).with(region.name()).warn(message);
+      throw new CTPException(Fault.BAD_REQUEST, message);
+    }
+  }
+
   private void checkCaseIsNotTypeCE(CaseContainerDTO caseDetails) throws CTPException {
```

Comment:
Also while here - should have spotted this previously if I did review the previous ticket.

Again method name does not reflect what it really does and the message being fixed makes this method only of use for the invalidate use case.

Please rename similarly, and pass in the message such that this method could be used in other use cases.

Reply:
will do